examples/echo.rs

```diff
@@ -52,7 +52,7 @@ async fn server(conns: usize) -> Result<()> {
         s.await.unwrap()?;
     }
 
-    client_handle.join().unwrap();
+    client_handle.join().unwrap().unwrap();
```

Comment:
This one remains because we now return whatever the user returns from the bootstrap future. In the case of this example, we return an error (that we were dropping on the floor before this patchset).

Reply:
yes that's fine, of course